client/src/router.tsx

```diff
@@ -121,6 +121,8 @@ registerRoute('workload/statefulset/:namespace/:name', params => <StatefulSet {.
 
 window.addEventListener('hashchange', onNavigate);
 
+window.onpopstate = onNavigate; // this is used by filter/sort query params to handle back button clicks
```

Comment:
For consistency, it would be nice to use `addEventListener` here instead (similar to the line above). Any objections?